### PR TITLE
Arm vis

### PR DESCRIPTION
**Pull request by qinyang-bao**
Merged with master

### CODE REVIEW

File: Workspace/src/rover_gui/include/rover_gui/gui.h [hunk 1]
```diff
@@ -3,7 +3,9 @@
 #define GUI_H
 
 #define UTM_POSE_TOPIC "/localization/pose_utm"
-#define ARM_POSE_TOPIC "/arm_interface/actual_joint_angles"
+#define ARM_POSE_TOPIC_ACTUAL "/arm_interface/actual_joint_angles"
+#define ARM_POSE_TOPIC_DESIRED "/arm_interface/desired_joint_angles"
```
Comment (wmmc88) [inline Workspace/src/rover_gui/include/rover_gui/gui.h]: All of these should not be #defines. It should probably be const strings objects

**Review by wmmc88 — CHANGES_REQUESTED** [review]

File: Workspace/src/rover_gui/include/rover_gui/gui.h (hunk 1 above)
Comment (qinyang-bao) [reply Workspace/src/rover_gui/include/rover_gui/gui.h]: should I leave the UTM_POSE_TOPIC or edit the mapwidget to include it as a const string?

Comment (wmmc88) [reply Workspace/src/rover_gui/include/rover_gui/gui.h]: Change that too

Comment (wmmc88) [reply Workspace/src/rover_gui/include/rover_gui/gui.h]: Tbh they should be launch file params that have a default value

Comment (qinyang-bao) [reply Workspace/src/rover_gui/include/rover_gui/gui.h]: can I edit the autonomy control widget as well? It also depends on these defines

Comment (wmmc88) [reply Workspace/src/rover_gui/include/rover_gui/gui.h]: Ye. Edit everything u need to

Comment (qinyang-bao) [reply Workspace/src/rover_gui/include/rover_gui/gui.h]: Alright got it, now all topics are in a cfg file and I made a launch file

**Pull request closed without merging**